ddtrace/span.py

```diff
@@ -163,12 +177,15 @@ def duration(self):
         """The span duration in seconds."""
```

Comment:
```suggestion
        # type: () -> int
        """The span duration in seconds."""
```

Reply:
This method is missing a return statement in case `duration_ns` is None, so I'll fix this in a separate PR and add the type hint then.